tests/__init__.py

```diff
@@ -8,7 +8,7 @@
 
 # This should match the major.minor version list in .circleci/generate_circleci_config.py
 # Patch version should always be 0
-supported_k8s_versions = ["1.21.0", "1.22.0", "1.23.0", "1.24.0"]
+supported_k8s_versions = ["1.22.0", "1.23.0", "1.24.0", "1.25.0"]
```

Comment:
Let's also leave 1.21 in here until EKS makes 1.21 EOL. We can remove this in March.